packages/rich-text-editor/src/vaadin-rich-text-editor-mixin.js

```diff
@@ -668,6 +728,36 @@ export const RichTextEditorMixin = (superClass) =>
       }
     }
 
+    /** @private */
+    _onColorClick() {
```

Comment:
Could align with the changes in `vaadin-rich-text-editor-popup-mixin.js` and use double-underscore for private. Same with the other new members in this file.

Reply:
Done.